mobile_verifier/src/lib.rs

```diff
@@ -21,8 +21,11 @@ pub mod telemetry;
 pub use settings::Settings;
 
 use async_trait::async_trait;
+use rust_decimal::Decimal;
 use std::error::Error;
 
+pub const MOBILE_SUB_DAO_ONCHAIN_ADDRESS: &str = "39Lw1RH6zt8AJvKn3BTxmUDofzduCM2J3kSaGDZ8L7Sk";
+
```

Comment:
This seems like something that should live in helium-lib? @madninja

Reply:
it already -is-.. just not hardcoded like this but iirc derived in dao.rs